scipy/sparse/tests/test_matrix_io.py

```diff
@@ -7,11 +7,11 @@
 
 import pytest
 from pytest import raises as assert_raises
-from numpy.testing import assert_equal, assert_
+from numpy.testing import assert_equal, assert_, assert_raises
```

Comment:
This `assert_raises` isn't needed, as we're getting it from the import on the previous line.

Reply:
I made the change but can you please explain the advantage of implementing it as you suggested?

Reply:
We're moving from nose to pytest. The version from `numpy.testing` is from nose, whereas the one imported on line 9 is from pytest. They do the same thing, but since we no longer require users to install nose to run the tests, we can't rely on `numpy.testing.assert_raises` being available.